ext/php7/serializer.c

```diff
@@ -555,7 +556,7 @@ void ddtrace_set_root_span_properties(ddtrace_span_t *span) {
                 if (query_str) {
                     query_string =
                         zai_filter_query_string((zai_string_view){.len = strlen(query_str), .ptr = query_str},
-                                                get_DD_TRACE_RESOURCE_URI_QUERY_PARAM_ALLOWED());
```

Comment:
This does not look right to me. With `DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED=true` the query string is sent unmodified by default to datadog, given the new `*` default. I think at this place we need the regex filter as well.

Reply:
Well, the default for `DD_TRACE_RESOURCE_URI_QUERY_PARAM_ALLOWED` is still an empty array, but I guess it also makes sense to add the regex here for the `*` case, so I'll do that.

Reply:
Right, I was confused with the different names, but yes by default the regex should apply too.